conda/env/env.py

```diff
@@ -264,6 +264,10 @@ def save(self):
         with open(self.filename, "wb") as fp:
             self.to_yaml(stream=fp)
 
+    def get_configuration_settings(self):
```

Comment:
This represents all the configuration parameters that a user is able to set in an `environment.yaml`